installer/pom.xml

```diff
@@ -80,7 +80,6 @@
 				<version>${tychoVersion}</version>
 				<configuration>
 					<includeAllDependencies>true</includeAllDependencies>
-					<includePackedArtifacts>false</includePackedArtifacts>
```

Comment:
Not sure to understand how it relates to $SUBJECT

Reply:
leftover after switch to tycho 2.5.0 and java11 that do not include packed jars anymore.